test/lib/npm.js

```diff
@@ -443,7 +417,7 @@ t.test('cache dir', async t => {
 
 t.test('timings', async t => {
   t.test('gets/sets timers', async t => {
-    const { npm, logs } = await loadMockNpm(t, { load: false })
+    const { npm, logs } = await loadMockNpm(t)
```

Comment:
How are we getting this same behavior w/o telling mock npm not to load?

Reply:
Another case of old tests being fake. The `load: false` might not have been necessary before. But now we have to tell npm to load, otherwise nothing will ever get written to the `stdout/stderr`.